_posts/2022-10-20-datafusion-13.0.0.md

```diff
@@ -0,0 +1,226 @@
+- Casting / coercion now happens logical planning (#3185) (#3396) (#3636)
+- More sophisticated expression analysis and simplification
+
+# Parquet
+ - Parquet reader can now read directly from remote parquet files on object storage (#2489) (#2677) (#3051)
+ - Experimental support for “predicate pushdown” that implements late materialization after filtering during the scan (we plan another blog post on this soon).
+ - Support reading directly from S3 via `datafusion-cli ` (#3631)
+
+# DataType Support
+- Support for `IN` LIST,  support, casting, etc for `Decimal`
+- Timestamps: `date_bin` built-in function (#3034), timestamp plus minus interval (#3110), TIME literal values (#3010)
+- Binary operations (AND, XOR, etc):  (#3037) (#1619) (#3420) (#3430)
+
+
+
+## Upcoming Work
+With the community growing and code accelerating, there is so much great stuff on the horizon. Some features we expect to land in the next few months:
+
+- [Complete Parquet Pushdown](https://github.com/apache/arrow-datafusion/issues/3462)
+- [Additional date/time support](https://github.com/apache/arrow-datafusion/issues/3148)
+- improve / make it easier to implement FlightSQL using DataFusion (TODO LINK)
+
+
+# Community Growth
+
+The DataFusion 9.0.0 and 13.0.0 releases consists of 433 PRs from 64 distinct contributors. This does not count all the great works that goes into the [arrow](https://crates.io/crates/arrow),  [parquet](https://crates.io/crates/parquet), and [object_store](https://crates.io/crates/object_store) libraries that the same community burtures.
+
+<!--
+$ git log --pretty=oneline 9.0.0..13.0.0 . | wc -l
+433
+
+$ git shortlog -sn 9.0.0..13.0.0 . | wc -l
+65
+-->
+
+
+# How to Get Involved
+
+
+Again, kudos to everyone in the community who contributed ideas, discussions, bug reports, documentation and code. It is exciting to be building something so cool together!
+
+If you are interested in contributing to DataFusion, we would love to
+have you join us on our journey of learning about state-of-the-art
+query processing!
+
+You can help by trying out DataFusion on some of your own data and
+projects and let us know how it goes or contribute a PR with
+documentation, tests or code. A list of open issues suitable for
+beginners is
+[here](https://github.com/apache/arrow-datafusion/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
+
+Check out our new [Communication Doc](https://arrow.apache.org/datafusion/community/communication.html) on more
+ways to engage with the community.
+
+## Appendix: Contributor Shoutout
+
+To give a sense of the number of people who contribute to this project regularly, we present for your consideration the following list derived from `git shortlog -sn 9.0.0..13.0.0 .` Thank you all again!
+
+```
+    87	Andy Grove
+    71	Andrew Lamb
+    29	Kun Liu
+    18	kmitchener
+    17	Wei-Ting Kuo
+    14	Yang Jiang
+    13	dependabot[bot]
+    12	Raphael Taylor-Davies
+    11	Batuhan Taskaya
+    11	Kirk Mitchener
```

Comment:
We should combine stats for `kmitchener` and `Kirk Mitchener`

Reply:
done